### DISCUSSION TITLE
Deleting specific chapters

### DISCUSSION THREAD

**Discussion by serrq** [PaulWoitaschek/Voice#1517] [Ideas]
### What happened?

I use Voice mainly to listen talk shows of which I only have the audio (I extract it from video).

Thus the chapters for me are nothing more than episodes, at the end of which I may wish to delete them once they have been listened to, leaving the remainder standing.

Therefore, I request the possibility of selectively eliminating the chapters.
In this regard, it would also be useful to add the "empty" folder function.

### Version

6.1.2

**Comment by bananabob** [comment]
I would like to add my support for this feature. I use Voice in exactly the same way. I listen to podcasts using the app and the ability to delete each "chapter" or episode.
